table/server/underdb/hive/src/main/java/alluxio/table/under/hive/HiveDatabase.java

```diff
@@ -340,7 +334,11 @@ public UdbTable getTable(String tableName, UdbBypassSpec bypassSpec) throws IOEx
         }
       }
 
-      mount(bypassSpec);
+      if (!mIsMounted) {
+        // TODO(bowen): bypass spec updates are ignored
+        mount(bypassSpec);
+        mIsMounted = true;
+      }
```

Comment:
This is problematic for now. If tables are dropped from outside of alluxio between two calls of getTable, then the path mapping info in the path translator will become stale.